MyCryptoHQ/MyCrypto
# Add assetOptions in LocalCache initialization

### Description

Adds default tokens to cache on cache initialization.

### Changes

* Added a method to add default tokens to cache on initialization
* Copied over config folder to `v2`.
* Added a method to write tokens from `ethereum-lists/tokens` repo to `v2/config` folder with a different structure in addition to the old folder (I included `name` field to conform to `AssetOption` type for the new folder's token data structure).

### Steps to Test

1. Delete localcache in browser, Hard reset page. Open dev tools in browser, navigate to *Application* tab, click on *Local Storage* then localhost and see if the tokens were added to the local cache under the MyCryptoCache key
